server/curfu/devtools/interpro.py

```diff
@@ -233,7 +233,6 @@ def get_protein_accessions(
                     cur_molecule_id = tmp_molecule_id
             else:
                 # TODO does this happen?
-                breakpoint()
```

Comment:
I didn't mean to actually include this change in this PR but I'm leaving it in bc there shouldn't be a breakpoint in production code, even if it's probably unreachable